From e2c2a9ce87f8c4dc5e6d0fb3d763a86e08b87426 Mon Sep 17 00:00:00 2001
From: Michael Simons <msimons@microsoft.com>
Date: Tue, 17 Aug 2021 21:15:31 +0000
Subject: [PATCH] Update assembly version attributes

Addresses errors from the generated AssemblyInfo.cs files:
error CS7034: The specified version string does not conform to the required format - major[.minor[.build[.revision]]]

TODO:  Need to understand why SB tarball is encountering these errors and the SB repo leg doesn't.
---
 build/common.project.props | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/common.project.props b/build/common.project.props
index 8acc4e0bb..f7bd1d472 100644
--- a/build/common.project.props
+++ b/build/common.project.props
@@ -190,8 +190,8 @@
 
   <PropertyGroup Condition=" '$(IsNetCoreProject)' == 'true' ">
     <!-- Assembly attributes for net core projects -->
-    <AssemblyVersion>$(SemanticVersion).$(PreReleaseVersion)</AssemblyVersion>
-    <FileVersion>$(SemanticVersion).$(PreReleaseVersion)</FileVersion>
+    <AssemblyVersion>$(SemanticVersion)</AssemblyVersion>
+    <FileVersion>$(SemanticVersion)</FileVersion>
     <InformationalVersion Condition="'$(BUILD_SOURCEVERSION)' == ''">$(SemanticVersion)$(PreReleaseInformationVersion)</InformationalVersion>
     <InformationalVersion Condition="'$(BUILD_SOURCEVERSION)' != ''">$(SemanticVersion)$(PreReleaseInformationVersion)+$(BUILD_SOURCEVERSION)</InformationalVersion>
     <Company>Microsoft Corporation</Company>
-- 
2.29.2

